core/plugins.Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Build image: Chainlink binary
+FROM golang:1.20-buster as buildgo
+RUN go version
+WORKDIR /chainlink
+
+COPY GNUmakefile VERSION ./
+COPY tools/bin/ldflags ./tools/bin/
+
+ADD go.mod go.sum ./
+RUN go mod download
+
+# Env vars needed for chainlink build
+ARG COMMIT_SHA
+
+COPY . .
+
+# Build the golang binaries
+RUN make install-chainlink
+RUN make install-solana
+RUN make install-median
+
+# Final image: ubuntu with chainlink binary
+FROM ubuntu:20.04
+
+ARG CHAINLINK_USER=root
```

Comment:
i know know a lot about docker, but it feels wrong to me to have the chainlink user be root. do we do this other places? if so, do you know why?

Reply:
IIUC this is only for CI